### ISSUE TITLE
Username and Password not redacted from logs in FFmpeg error logs

### ISSUE THREAD

**Issue by sdwilsh** [roflcoopter/viseron#294]
While Username and passwords are redacted from logs in the FFmpeg command, if there is an error, they are not redacted.

```
[2021-10-28 12:30:59] [ERROR   ] [viseron.camera.stream.front_porch] - Retrying None in 2.0 seconds as it raised FFprobeTimeout: FFprobe command ffprobe -hide_banner -loglevel error -print_format json -show_error -show_streams rtsp://{non-redacted username}:{non-redacted password}@10.67.0.10:554/h264Preview_01_sub timed out after 15s.
```

**Comment by roflcoopter** [comment]
Will fix in #306 when its released

**Comment by roflcoopter** [comment]
Closed in v2
